contrib/dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
+MAINTAINER Taihsiang Ho <tai271828@gmail.com>
+RUN apt-get -qq update
+RUN apt-get -qqy install g++ liblapack-dev git wget
+RUN useradd -m solvcon
+USER solvcon
+ENV HOME="/home/solvcon"
+ENV PATH="$HOME/miniconda/bin:$PATH"
+ENV DOCKER_SCSRC="$HOME/solvcon"
+RUN cd $HOME; wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+RUN bash $HOME/miniconda.sh -b -p $HOME/miniconda
+RUN conda config --set always_yes yes --set changeps1 no
+RUN conda update -q conda
+RUN cd $HOME; git clone https://github.com/solvcon/solvcon.git
+RUN ${DOCKER_SCSRC}/contrib/conda.sh
+RUN cd ${DOCKER_SCSRC}; /bin/bash -c "source activate; python setup.py build_ext --inplace"
```

Comment:
This ``bash -c`` looks not so graceful. docker uses ``sh`` instead of ``bash`` by default and no ``source`` command could be used.